bundles/org.openhab.binding.millheat/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:millheat:account">
+		<parameter name="username" type="text">
+			<label>Username</label>
+			<description>Your Millheat app username (email)</description>
+			<required>true</required>
+		</parameter>
+
+		<parameter name="password" type="text">
+			<label>Password</label>
+			<description>Your Millheat app password</description>
+			<required>true</required>
```

Comment:
use parameter (element is deprecated: https://www.eclipse.org/smarthome/documentation/development/bindings/xml-reference.html)

Reply:
Fixed